packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -132,7 +132,7 @@ class MouseTracker extends ChangeNotifier {
   /// layer tree.
   ///
   /// If the associated layer is not removed, and receives a hit, then
-  /// [sendMouseNotifications] will assert the next time it is called.
+  /// [_sendMouseNotifications] will assert the next time it is called.
```

Comment:
I don't think mentioning a private method is necessary. you can just mention an assert error will be thrown when associated layer is not removed and receives a hit.

Reply:
Good call, I'll fix it.